sematic/resolvers/tests/test_log_streamer.py

```diff
@@ -79,7 +85,7 @@ def test_ingested_logs_uncaught():
             uploads.append(list(upload))
         upload_number += 1
 
-    assert len(uploads) == 1
+    assert len(uploads) >= 1
```

Comment:
Flake mode: sometimes the main process doesn't complete before the uploader has finished its initial upload. So you end up with more uploads than just the final one. Now we're ok if extras happen. We can still ensure that the process finished before waiting a full upload interval using the `assert time.time() - started < upload_interval` statement. Will this flake? Not unless the OS fails to schedule the main process for 100 consecutive seconds.